pkg/operation/botanist/component/etcd/etcd.go

```diff
@@ -514,21 +514,20 @@ func (e *etcd) Deploy(ctx context.Context) error {
 			Quota:                   &quota,
 		}
 
-		if e.nodeSpread {
-			e.etcd.Spec.Etcd.PeerUrlTLS = &druidv1alpha1.TLSConfig{
-				TLSCASecretRef: druidv1alpha1.SecretReference{
-					SecretReference: corev1.SecretReference{
-						Name:      etcdPeerCASecretName,
-						Namespace: e.namespace,
-					},
-					DataKey: pointer.String(secretutils.DataKeyCertificateBundle),
-				},
-				ServerTLSSecretRef: corev1.SecretReference{
-					Name:      peerServerSecretName,
+		e.etcd.Spec.Etcd.PeerUrlTLS = &druidv1alpha1.TLSConfig{
```

Comment:
If Peer TLS is changed (enabled) for existing etcd cluster then it will result in 2 restarts of the etcd cluster. This is unfortunately a requirement for etcd. Also See https://github.com/gardener/etcd-druid/issues/416